client/src/modules/shipment/create-shipment.js

```diff
@@ -127,6 +127,10 @@ function CreateShipmentController(
   };
 
   vm.configureItem = function configureItem(row, lot) {
+    if (lot.__is_asset === 1) {
```

Comment:
I had tried isAsset() and it did not work.  When I reexamined it, I realized that the isAsset() test in LotItemService was faulty because when we use the is_asset as a checkbox element in the html, its values are 1 and 0.  So the isAsset() test that did val === true did not work properly.  I changed it to use the !! truth value and it works now.  So I was able to change this to isAsset().